recipes/tinycbor/0001-Add-CMake-support.patch

```diff
@@ -0,0 +1,209 @@
+From e0fcc34f29633e8d4213581ad1c40575f9d2a0f1 Mon Sep 17 00:00:00 2001
+From: Samuel Debionne <samuel.debionne@esrf.fr>
+Date: Wed, 23 Aug 2023 16:13:36 +0200
```

Comment:
Have you opened a PR to upstream this patch? Does this CMake build system generate artifacts that are named the same as the ones generated by makefiles? For example, sometimes the DLL name on windows might have/not have the lib prefix.

Reply:
I am not sure the original build system supports Windows, I'll need to check.